sinks/api/http/endpoint_test.go

```diff
@@ -996,21 +1005,21 @@ func TestValidateSink(t *testing.T) {
 	}
 }
 
-func TestOmitPasswords(t *testing.T) {
-	cases := map[string]struct {
-		inputMetadata    types.Metadata
-		expectedMetadata types.Metadata
-	}{
-		"omit configuration with password": {
-			inputMetadata:    types.Metadata{"user": 387157, "password": "s3cr3tp@ssw0rd", "url": "someUrl"},
-			expectedMetadata: types.Metadata{"user": 387157, "password": "", "url": "someUrl"},
-		},
-	}
-
-	for desc, tc := range cases {
-		t.Run(desc, func(t *testing.T) {
-			metadata := omitSecretInformation(tc.inputMetadata)
-			assert.Equal(t, tc.expectedMetadata, metadata)
-		})
-	}
-}
+//func TestOmitPasswords(t *testing.T) {
```

Comment:
for what reason we are skipping this test?

Reply:
This is no longer applicable in this package, since now it has the dependency of the backend, I am looking for someplace else which is more suitable, to not add cyclic dependencies.